src/Repository/EventRegistrationRepository.php

```diff
@@ -104,6 +105,19 @@ public function findUpcomingAdherentRegistrations(string $adherentUuid): array
         ;
     }
 
+    public function findActivityUpcomingAdherentRegistrations(string $adherentUuid, int $page = 1, int $limit = 5)
+    {
+        self::validUuid($adherentUuid);
```

Comment:
why do not pass Adherent instance like for others?

Reply:
I based this method on this one https://github.com/EnMarche/en-marche.fr/pull/4640/files/54859de557df570e5f1960f43241bdd8b083697c#diff-39f09d2ecd7d2f1787e990441abd3a7fL86
just above